### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by lisandroseia**
In this branch I implemented:
A classroom class, that holds many students
Classroom in each student object
A book class, that holds many rentals
A rental class, that holds a book and a person
Added rentals to the person class, which holds the rentals for each person's object
Kindly review, and let me know if something is wrong!

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rent)
+    @rentals.push(rent)
+  end
```
Comment (Dipeshtwis) [inline book.rb]: - [ ] In The add_rentals method, you have to create a rental passing all the required parameters. Here, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(person, date)
  Rental.new(date, person, self)
end
```

File: person.rb [hunk 2]
```diff
@@ -26,4 +27,8 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def add_rentals(rent)
+    @rentals.push(rent)
+  end
```
Comment (Dipeshtwis) [inline person.rb]: - [ ] In The add_rentals method, you have to create a rental passing all the required parameters. Here, the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```ruby
def add_rental(book, date)
  Rental.new(date, self, book)
end
```

File: student.rb [hunk 3]
```diff
@@ -1,11 +1,18 @@
 require_relative 'Person'
 
+attr_accessor :classroom
+
```
Comment (Dipeshtwis) [inline student.rb]: - [ ] We just have to set the getter for the `classroom` instance variable. Please update it.

Hint:
`attr_reader :classroom`

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @lisandroseia, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @lisandroseia, 

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

- Well done!

Charlie Brown

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**